trunner/runners/__init__.py

```diff
@@ -9,9 +9,11 @@
 # Authors: Mateusz Niewiadomski
 #
 
+from .ARMV7M7Runner import ARMV7M7Runner
 from .HostRunner import HostRunner
 from .QemuRunner import QemuRunner
 from .IMXRT106xRunner import IMXRT106xRunner
+from .IMXRT117xRunner import IMXRT117xRunner
 from .STM32L4Runner import STM32L4Runner
 
-__all__ = ['HostRunner', 'QemuRunner', 'IMXRT106xRunner', 'STM32L4Runner']
+__all__ = ['ARMV7M7Runner', 'HostRunner', 'QemuRunner', 'IMXRT106xRunner', 'IMXRT117xRunner', 'STM32L4Runner']
```

Comment:
I think `ARMV7M7Runner` is internal class and should not be exported by `runners` module by default